app/models/tag.rb

```diff
@@ -40,6 +42,20 @@ class Tag < ApplicationRecord
   scope :archived, -> { where(type: ArchivedTag.to_s) }
 
   after_update_commit ->(tag) { EventBus.publish(:tag_renamed, tag) if previous_changes.key?("name") }
+  after_update_commit ->(tag) do
+    tag.filters.each do |filter|
+      filter.user_filter_visibilities.update_all(visible: false)
+    end if previous_changes.key?("visible") && !tag.visible
+  end
+
+  after_create ->(tag) do
+    TagFilter.create!(
+      tenant: tag.tenant,
+      author: tag.owner,
+      name: tag.name,
+      tag:,
```

Comment:
Tu nieco nechyba?

Reply:
Narazas na Shorthand Hash Syntax?

Reply:
To poznam, ale preco je tag: nil?

Reply:
No tam nie je `nil`, ved zapis `tag:,` je skratena verzia pre `tag: tag`. To nie je to iste ako `tag: nil`

Reply:
Ah shit, tak toto nepoznam a vobec sa mi to nepaci :) "write only code"

Reply:
Za mna je to super vychytavka, priklad
`some_method(a: a, b: b, c: c, d: d + 4)` 
je skratene 
`some_method(a:, b:, c:, d: d + 4)`